docs/DeveloperGuide.adoc

```diff
@@ -358,14 +358,22 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *`   |Forgetful student | Set reminders for upcoming events | Will remember to attend them
 
+|`* * *` |Student | Add tasks to study plan | Can add study tasks to my study plan
+
+|`* * *` |Student | Delete tasks on study plan | Can delete study tasks if I don't need it anymore
+
+|`* * *` |Student | Auto-generate event from a task | If I feel like I want to do a study task today, I can generate an event on today's schedule, or any day.
+
+|`* *`   |Student | View statistics of my task progress | Can see how much I've done for each tasks on my study plan.
+
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `NOVA` and the *Actor* is the `student`, unless specified otherwise)
+(For all use cases below, the *System* is the `nova` and the *Actor* is the `student`, unless specified otherwise)
```

Comment:
There is no need to change to lowercase for NOVA. Lowercase nova is just used for our package because there is the checkstyle.